dist/js/patient.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+
+var _observations = require('./observations');
+
+var _observations2 = _interopRequireDefault(_observations);
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```

Comment:
If we are following the style guide : https://github.com/airbnb/javascript
Then we need to make the "}" go on a new line, unless this single line format is preferred by the ES6 standard.